web-ui/src/main/resources/catalog/components/catalog/CatalogService.js

```diff
@@ -421,6 +421,7 @@
           isXLinkEnabled: 'system.xlinkResolver.enable',
           isSelfRegisterEnabled: 'system.userSelfRegistration.enable',
           isFeedbackEnabled: 'system.userFeedback.enable',
+          isInspireEnabled: 'system.inspireValidation.enable',
```

Comment:
This is in a comment section, but the value is not fine, should be the same as in line 440: `isInspireEnabled: 'system.inspire.enable',`